### PR TITLE
Change paradox to not immediately be an antag

### PR DESCRIPTION
**Pull request by Stop-Signs**
## About the PR
Remove the antag pass on Paradox Anomalies. They are no longer allowed to just go and murder their double with no RP.

## Why / Balance
Recent experience where my paradox ignored my friendship and instead meatspiked me using the free antag pass that is inherent to the role. It genuinely ruined my night and hopefully this PR will save that from happening to others

## Technical details
N/A

## Media
N/A

## Requirements

- [ x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [ x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

### CODE REVIEW

**Review by sheepism1 — COMMENTED** [review]
I don't like this PR very much!
 
 Paradox clones are meant to either replace their targets  _Mandela Catalogue_ style or befriend them. I think it takes all the fun out of this to force Paradoxes to be happy with this new universe with 2 of them. Great idea for a rule change about NRP murder though!

**Comment by sheepism1** [comment]
Honestly taking the choice out of "are they a friend or do I replace them" and forcing them to be a crew-aligned ghost role, or at most a free agent, is not fun for any of the OTHER people who roll paradox just because one person got RR'd by a ghost role.. *doing their objective.*

**Comment by sheepism1** [comment]
Honestly this is like forcing a traitor to be a friendly role with 20tc because one gibbed you with a throngler when they had the objective to do so.

I'd get it if they were a clone of another guy and you were just in the wrong place wrong time.

**Comment by sheepism1** [comment]
An ahelp might have done it.

**Comment by FieldCommand** [comment]
I am for this change. Every time I have had a paradox of myself, I have been killed 8 times out of 10. I have seen the paradox of others killing the original. like 80% they, "Oh, I get a free kill pass." There is no effort or anything. When the RP is good, its glorious. And at times when they kill me or other originals, they don't even try to RP it out. I once was a prisoner, it was evac time, and the prisoner showed up out of nowhere, killed me with a toolbox, not saying a thing, killed a fellow inmate, as the HoS watched, and then proceeded to hide in a crate as if the world did not exist.

**Comment by Stop-Signs** [comment]
> An ahelp might have done it.

I actually did ahelp this round and had a good conversation with the admin. They said that on a technicallity the paradox was allowed to do what they did as they had an "antag pass". this PR doesnt entirely negate the ability to replace your double, it simply removes the antag objective too and some of the wording, by all means if your clone escalates it you can kill them and replace them. however the goal of this PR is to prevent round removal just for existing as you, and instead promote healthier RP between the body doubles. 

furthermore as a clone of someone you should have similar mannerisms as them, i would never kill someone just because they look and sound like me, however since the person playing my clone had an "antag pass", they were permitted to abandon all likeness and just murder.

**Comment by Stop-Signs** [comment]
> Honestly taking the choice out of "are they a friend or do I replace them" and forcing them to be a crew-aligned ghost role, or at most a free agent, is not fun for any of the OTHER people who roll paradox just because one person got RR'd by a ghost role.. _doing their objective._

furthermore i actually saw the exact same thing happen on an apo round this morning. paradox without words just straight up started killing their orignal. As previously stated the goal is not to remove the possibility that you replace them, however it is to disincentivize straight up murder by removing it as a _goal_ and make it a _possibility_.

**Comment by sheepism1** [comment]
fair

**Comment by dge21** [comment]
so what is the goal of the paradox now? 

Just, hang out and be a funny clone?

**Comment by deltanedas** [comment]
objective wise paradox would now be a personal paramedic

**Comment by perryprog** [comment]
So I think there are a few interesting things here worth discussing. (Note I'll be talking about my personal experience on peri which I know can have a different atmosphere than Apo.)

- I have definitely seen paradox anomalies on a few occasions (I want to say two or maybe three times) go out and just immediately try kill their original, without any attempt at interaction. I do agree that this is problematic, but I'm not sure if there's an easy way to resolve this that isn't explicitly stating... don't be a murderbone if you don't have antagonist status plus a good enough reason for it. I feel like this should already be the case, though, as this _is_ an MRP server: even when you're an antagonist you should be playing within the context of roleplay, and if you aren't, then that should be an admin issue.
- While I haven't been too attentive in watching these moments, I have been on rounds where there _was_ roleplay heavy interaction between the paradox and the original that ended up coming to fisticuffs in a way that was well-justified by the roleplay. This is cool; roleplay-based conflict is good.
- The _vast majority_ of paradox anomalies I've seen go for the befriend route, and that's what I always end up choosing (having been paradox about four times now). The way I personally approach the roleplay is I will drop subtle hints that I have some sort of superiority complex over the original, or perhaps a drive to surpass (hint hint, replace) them. Thus, depending on how future interactions go, it's partly up to the original to either work to try to resolve this tension or to aggravate it instead. (What has happened so far is that by the end of the round we end up being reluctant friends.)

By having the _option_ of being able to go and just kill my clone, there's an implicit (somewhat metagamey) threat that I could do something that'd be uncalled for—namely, attempted murder. Removing the dichotomy between replacement versus friendship, I feel, would greatly diminish this dynamic in a way that would make the role much less interesting.

TL;DR: If people are using paradox anom as an NRP excuse to just kill someone then that's bad and should be resolved via rules/admin intervention and not by limiting the agency of people who play paradox in order to take advantage of the "will they/won't they" threat.

**Comment by Stop-Signs** [comment]
> TL;DR: If people are using paradox anom as an NRP excuse to just kill someone then that's bad and should be resolved via rules/admin intervention and not by limiting the agency of people who play paradox in order to take advantage of the "will they/won't they" threat.

I totally agree. if i could PR a rule change i would, but i cant, so removing the "antag pass" is the only real way I can fix this issue.

**Comment by deltanedas** [comment]
> They are no longer allowed to just go and murder their double with no RP.

this implies they would still be allowed to murder them *with* rp, but since you removed antag status that would be selfantag

killing someone out of the blue without any rp is an admin issue, and applies to any antag that can be asked to kill someone (traitor suicide bombing with no interaction of any kind, anyone?).
for the nrp types they still dont greentext because of the kill-protect dual objectives, thats the whole point of it

this being merged would basically be the opposite of how paradox was in legacy, ooc you know they always want to kill the real one, instead now you know they arent allowed to kill so you never have to worry about anything which is just sad
